_data/amritsrivastava/amritsrivastava.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Amrit Srivastava
+institution: Jaipur Engineering College and Research Center
+profile_pic: amrit.jpg
+quote: 'Be not afraid of growing slowly, be afraid of standing still.'
+github_user: amritsrivastava
+---
```

Comment:
Can you please remove the extra 2 new lines?

Reply:
Thanks @Mohammed-Shoaib for the review. 
Lines removed.